HelloImKevo/AndroidPresentationDisplayDemo
# Integrate Android Studio Dolphin AdMob project boilerplate to show AdMob test ads

## Test Plan:
- Launch the app.
- Click on the AdMob option in the side navigation drawer.
- Confirm Toast Messages show.
- Click on the "NEXT LEVEL" button.
- Confirm that full screen test ads are shown.

## Live Action:
![android-admob-01](https://github.com/HelloImKevo/AndroidPresentationDisplayDemo/assets/4968350/2f0fee21-2119-42aa-a80c-81ebb9158782)

![android-admob-02](https://github.com/HelloImKevo/AndroidPresentationDisplayDemo/assets/4968350/652d2144-a534-48f1-866b-c89f859457ba)
